src/target/riscv/riscv-013.c

```diff
@@ -502,6 +507,7 @@ static int dmi_read(struct target *target, uint32_t *value, uint32_t address)
 			LOG_ERROR("failed read from 0x%x, status=%d", address, status);
 			return ERROR_FAIL;
 		}
+		usleep(100000);
```

Comment:
both of these changes were attempting to avoid the problem when you have a slow simulator, you get a massive log file full of not-too-useful error messages that becomes a problem in and of itself.